thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionMigrationResource.java

```diff
@@ -271,6 +271,9 @@ private String getTimezone(AnomalyFunctionDTO functionDTO) {
   }
 
   private String getBucketPeriod(AnomalyFunctionDTO functionDTO) {
+    if (functionDTO.getBucketSize() == 1 && functionDTO.getBucketUnit().equals(TimeUnit.DAYS)){
+      return "P1D";
```

Comment:
Can be more general to Period.days(functionDTO.getBucketSize()).toString()

Reply:
updated